src/exec/processor.rs

```diff
@@ -103,8 +103,8 @@ impl MapCompiler {
 
     let const_exprs = izip!(0..exprs.len(), exprs).filter(|e| match *e.1.kind() {ExprKind::Const(_) => true, _ => false});
     let nonconst_exprs = izip!(0..exprs.len(), exprs).filter(|e| match *e.1.kind() {ExprKind::Const(_) => false, _ => true});
-    
-    let schema_map = schema.to_map();    
+
```

Comment:
The changes for spaces may be affected by some editor configurations. We may need to decide some coding standard rules and editor configurations.